### ISSUE TITLE
CuckooL No activity being reported for Behaviorial analysis on Ubuntu Host and centos 7 guest

### ISSUE THREAD

**Issue by assadabbas** [cuckoosandbox/community#407]
Hi,

I want to execute elf's on centos 7 and want to analyze the sample using cuckoo sandbox. I have installed cuckoo 2.0.5 on Ubuntu host and Centos 7 as guest. Followed all the instructions mentioned at https://github.com/cuckoosandbox/cuckoo/blob/master/docs/book/installation/guest/linux.rst . When i submit an elf sample it gets executed on client but no activity is reported for dynamic analysis. The sample creates some files then execute these files. None of the activity is reported in Behavioral analysis, dropped files section. No error/warning appears in logs.

Any help would be grateful.

**Comment by doomedraven** [comment]
logs?

**Comment by doomedraven** [comment]
all logs, it is in `$CWD/storange/analyses/id/`

**Comment by assadabbas** [comment]
Hi,

I have attached all the logs.

**Comment by doomedraven** [comment]
False

**Comment by assadabbas** [comment]
I cannot understand "False".

**Comment by doomedraven** [comment]
you didn't attach the log

**Comment by assadabbas** [comment]
Can't you find logs.zip?

**Comment by assadabbas** [comment]
I am attaching the logs.zip file again.

**Comment by doomedraven** [comment]
false, go to thread and upload them manually, and plz do that next time to save us time if you want get help asap

**Comment by asadzaigum** [comment]
I dont know why you are not able to find logs attached to email thread. Anyway i am attaching the logs here now.
[logs.zip](https://github.com/cuckoosandbox/community/files/1893156/logs.zip)

**Comment by doomedraven** [comment]
email which you send isn't sent to me, it sent to gh, and it cut it off. stap.log contains behaviour, and there no errors in logs. so no idea. which is your cuckoo version?

**Comment by assadabbas** [comment]
2.0.5

**Comment by doomedraven** [comment]
latest is `Cuckoo (2.0.5.3)                  - Automated Malware Analysis System`

**Comment by doomedraven** [comment]
if you see your logs which you send me, your stap.log has the content

**Comment by doomedraven** [comment]
you always can use https://linux.huntingmalware.com/ with all supported arches

**Comment by assadabbas** [comment]
Thanks for sharing the link. Can you explain which OS are supported in your
setup. As I would like to submit an elf to a centos 7. Is it supported in
your setup?

**Comment by doomedraven** [comment]
nah, only ubuntu, maybe one day i will extend the list but right now it works just fine with ubuntu for all arches, why do you need specially centos?
